tomee/tomee-microprofile/mp-common/pom.xml

```diff
@@ -27,7 +27,61 @@
   <artifactId>mp-common</artifactId>
   <name>TomEE :: TomEE :: MicroProfile Common</name>
 
+  <dependencyManagement>
```

Comment:
same comment as above

Reply:
i believe this is not a duplicate, there are artifacts for API and more artifacts for Impl.